Update translation API costs in the README file (#3544)
responsible-ai-collaborative/aiid: site/docs/i18N.md

The translate API charges ~20USD per million characters and can translate to 111 languages.

> At the time of writing, there are 1033 Incidents and 4908 Reports, each incident consisting of ~373 characters, and each report consisting of ~5251 characters.

Reviewer: ```suggestion
As of May 2025, there are 1033 Incidents and 4908 Reports, on average consisting of ~373 characters, and each report consisting of ~5251 characters.
```

#### Incident Statistics
- Total incidents: 1033
- Average characters per incident: 373

Total incident characters: `1033 incidents * 373 characters = 385.309 characters`

> Cost of translate all incidents to 4 languages (en, es, fr, ja): `385.309 characters / 1 million * USD 20 * 4 languages = ~USD 31`

Reviewer: ```suggestion
Cost to translate all incidents to 4 languages (en, es, fr, ja): `385.309 characters / 1 million * USD 20 * 4 languages = ~USD 31`
```

#### Report Statistics
- Total reports: 4908
- Average characters per report: 5251

Total report characters: `4908 reports * 5251 characters = 25.771.908 characters`

> Cost of translate all reports to 4 languages (en, es, fr, ja): `26 million characters / 1 million * USD 20 * 4 languages = ~USD 2080`

Reviewer: ```suggestion
Cost to translate all reports to 4 languages (en, es, fr, ja): `26 million characters / 1 million * USD 20 * 4 languages = ~USD 2080`
```
